src/modules/linkedin.js

```diff
@@ -7,15 +7,15 @@
 			oauth: {
 				version: 2,
 				response_type: 'code',
-				auth: 'https://www.linkedin.com/uas/oauth2/authorization',
-				grant: 'https://www.linkedin.com/uas/oauth2/accessToken'
+				auth: 'https://www.linkedin.com/oauth/v2/authorization',
+				grant: 'https://www.linkedin.com/oauth/v2/accessToken'
 			},
 
 			// Refresh the access_token once expired
 			refresh: true,
 
 			scope: {
-				basic: 'r_basicprofile',
+				lite: 'r_liteprofile',
```

Comment:
I thought that `basic` was sort of a reserved keyword in Hello? Doesn't it get included by default if it exists? It seems like the behavior most people would want is for this to be `basic: 'r_liteprofile'` so basic continues to 'just work'.

Reply:
Oh yeah, I think you're right. Good catch! Thanks.